README.md

```diff
@@ -1,53 +1,67 @@
 # Description
-This module allows you to execute OQL queries from a microflow.It allows setting named parameters which will be properly escaped.
+This module allows you to execute OQL queries from a microflow. It allows setting named parameters which will be properly escaped.
 
 # Typical usage scenario
 Query data from the Mendix database and the result will be mapped to a Mendix entity.
 The example below is part of the Example module within the project, located on the GitHub repository.
 
-Considering the following domain model:
+As an example, one can execute a query obtaining all objects of `ExamplePerson` as a table. The rows within this table can be converted to non-persistent entities for further usage. A `ExamplePersonResult` non-persistent entity is modeled.
 
-One can execute a query obtaining all objects of ExamplePerson as a table. The rows within this table can be converted to non persistent entities for further usage. Therefore a ExamplePersonResult non persistent entity is modeled.
-
-Parameters can be used using their corresponding actions (for each data type there's one activity defined:
-
- 
-
-All parameters should be added before executing an OQL query.
-
- 
+Parameters can be used using their corresponding actions. For each data type there's one activity defined. All parameters should be added before executing an OQL query.
 
 Adding an parameter requires:
+* The name of the parameter, without a $ (e.g. Gender).
+* The value. 
 
-The name of the parameter, without a $ (e.g. Gender).
-The value.
 Executing an OQL query requires:
-
-The OQL statement.
-A resulting entity (e.g. OQLExample.ExamplePersonResult).
-Amount.
-Offset.
+* The OQL statement
+* A resulting entity (e.g. OQLExample.ExamplePersonResult)
```

Comment:
Let's format it as `OQLExample.ExamplePersonResult` here and below and consistently add module name to the entity name